### PR TITLE
sync with latest linguist version

### PR DESCRIPTION
**Pull request by smola**

### CODE REVIEW

**Comment by smola** [comment]
I recommend to review with `git diff -b` instead of GitHub UI. There is just too much noise from indentation length changes.

**Comment by dennwc** [comment]
@smola Consider [this](https://github.com/src-d/enry/pull/152/files#diff-7ff3ef94402341c21320e8324a2cd4d6) and [this](https://reviewable.io/reviews/src-d/enry/152/data/alias.go)

File: data/content.go [hunk 1]
```diff
@@ -494,13 +510,7 @@ var (
...
 	gs_Gosu_Matcher_0                      = substring.Regexp(`(?m)^uses java\.`)
 	h_ObjectiveDashC_Matcher_0             = substring.Regexp(`(?m)^\s*(@(interface|class|protocol|property|end|synchronised|selector|implementation)\b|#import\s+.+\.h[">])`)
-	h_CPlusPlus_Matcher_0                  = substring.Regexp(`(?m)^\s*#\s*include <(cstdint|string|vector|map|list|array|bitset|queue|stack|forward_list|unordered_map|unordered_set|(i|o|io)stream)>`)
-	h_CPlusPlus_Matcher_1                  = substring.Regexp(`(?m)^\s*template\s*<`)
-	h_CPlusPlus_Matcher_2                  = substring.Regexp(`(?m)^[ \t]*try`)
-	h_CPlusPlus_Matcher_3                  = substring.Regexp(`(?m)^[ \t]*catch\s*\(`)
-	h_CPlusPlus_Matcher_4                  = substring.Regexp(`(?m)^[ \t]*(class|(using[ \t]+)?namespace)\s+\w+`)
-	h_CPlusPlus_Matcher_5                  = substring.Regexp(`(?m)^[ \t]*(private|public|protected):$`)
-	h_CPlusPlus_Matcher_6                  = substring.Regexp(`(?m)std::\w+`)
+	h_CPlusPlus_Matcher_0                  = substring.Regexp(`(?m)CPlusPlusRegex`)
```
Comment (dennwc) [inline data/content.go]: This certainly doesn't work

Comment (smola) [reply data/content.go]: Nice catch. Something needs updating in the parser.

**Comment by smola** [comment]
It is also worrying that we do not catch these things with our tests.

**Comment by smola** [comment]
Dropping this PR, but filed an issue for it: https://github.com/src-d/enry/issues/155

**Pull request closed without merging**